product-service/handlers/createProduct.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+const {Client} = require('pg');
+
+const {PG_HOST, PG_PORT, PG_DATABASE, PG_USERNAME, PG_PASSWORD} = process.env;
+
+const dbOptions = {
+    host: PG_HOST,
+    port: PG_PORT,
+    database: PG_DATABASE,
+    user: PG_USERNAME,
+    password: PG_PASSWORD,
+    ssl: {
+        rejectUnauthorized: false
+    },
+    _connectionTimeoutMillis: 5000
+}
+
+const validHeaders = {
+    'Access-Control-Allow-Origin': '*',
+    'Access-Control-Allow-Credentials': true,
+}
+
+export const createProduct = async (event) => {
+    const client = new Client(dbOptions);
+
+    // Log incoming event
+    console.log('event: ', event);
+
+    let body = event.body;
+
+    let {
+        title = '',
+        description = '',
+        price = '',
+        count = 0
+    } = body;
+
+    // Check if product data is invalid
+    if (title === '' || description === '' || price === '' || count === '') {
```

Comment:
Could you please use some out-of-the-box solution such as ajv, joi, and so on for validation